actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -195,8 +195,15 @@ def request_method_symbol
     # Returns the original value of the environment's REQUEST_METHOD,
     # even if it was overridden by middleware. See #request_method for
     # more information.
-    def method
-      @method ||= check_method(get_header("rack.methodoverride.original_method") || get_header("REQUEST_METHOD"))
+    #
+    # For debugging purposes, when called with arguments this method will
+    # fallback to Kernel#method
+    def method(*args)
+      if args.any?
```

Comment:
`any?` isn't a good substitute for `!empty?`:

```ruby
>> [false].any?
=> false
```

Reply:
didn't know that, thanks